Scripts/script-CommonServerPython.yml

```diff
@@ -1,6 +1,7 @@
 commonfields:
   id: CommonServerPython
   version: -1
+  "fromVersion": "3.6.0"
```

Comment:
this will cause error for all content release until version 3.6.
this file will be missed from the content.zip and crash all (or at least most of ) python integrations/scripts.

possible solutions:
1) duplicate the original file with "toversion"
2) remove this label and apply the change for all content release
** however other code may depend on the current name convention

@dorsha @anara123